coriolis/tests/scheduler/test_scheduler_utils.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2024 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+from coriolis import constants
+from coriolis import exception
+from coriolis.scheduler import scheduler_utils
+from coriolis.tests import test_base
+
+
+class CoriolisTestException(Exception):
+    pass
+
+
+class SchedulerUtilsTestCase(test_base.CoriolisBaseTestCase):
+    """"Test suite for the Coriolis scheduler utils package."""
```

Comment:
NIT: docstring has too many double-quotes (4 instead of 3)
```suggestion
    """Test suite for the Coriolis scheduler utils package."""
```